presto-main/src/main/java/com/facebook/presto/operator/SortedPositionLinks.java

```diff
@@ -151,13 +152,10 @@ public Factory build()
                 }
             }
 
-            return lessThanFunction -> {
-                checkState(lessThanFunction.isPresent(), "Using SortedPositionLinks without lessThanFunction");
```

Comment:
please reintroduce the check in new code checking if provided List is not empty.